src/test/java/com/wikia/webdriver/testcases/fandom/SearchInputTest.java

```diff
@@ -0,0 +1,50 @@
+package com.wikia.webdriver.testcases.fandom;
+
+import com.gargoylesoftware.htmlunit.javascript.host.URL;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.fandom.FandomTestTemplate;
+import com.wikia.webdriver.elements.fandom.components.SearchInput;
+import org.openqa.selenium.By;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.By'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)